CONTRIBUTING.md

```diff
@@ -47,20 +50,24 @@ Once the pull request is approved, a team member will take care of merging.
 
 ## Setup environment
 
+Python 3.7 or later is required.
+
 Setting up your KerasNLP development environment requires you to fork the
-KerasNLP repository, clone the repository, install dependencies, and execute
-`python setup.py develop`.
+KerasNLP repository, clone the repository, create a virtual environment, and 
+install dependencies.
 
 You can achieve this by running the following commands:
 
 ```shell
 gh repo fork keras-team/keras-nlp --clone --remote
 cd keras-nlp
+python -me venv ~/keras-nlp-venv
```

Comment:
I think you added the -e in the wrong place? I can just fix this up as I land. This should go on the pip install comment.